include/vcpkg/base/messages.h

```diff
@@ -288,16 +346,17 @@ namespace vcpkg::msg
     {                                                                                                                  \
         using is_message_type = void;                                                                                  \
         static constexpr ::vcpkg::StringLiteral name = #NAME;                                                          \
-        static constexpr ::vcpkg::StringLiteral extra_comment = COMMENT;                                               \
+        static constexpr StringArray extra_comment = COMMENT;                                                          \
         static constexpr ::vcpkg::StringLiteral default_format_string = __VA_ARGS__;                                   \
         static const ::size_t index;                                                                                   \
+        static constexpr StringArray example_str = vcpkg::msg::detail::get_examples ARGS;                              \
```

Comment:
This calls the function `get_examples`. The `()` part is hidden inside the macro parameter.